collections/ansible_collections/massopencloud/esi/roles/network/meta/argument_specs.yaml

```
@@ -0,0 +1,15 @@
+argument_specs:
+  main:
+    options:
+      network_state:
+        type: "str"
+        required: true
```

```suggestion
        default: present
```
it's quite common to default to `present` in ansible world

---

Ah, thanks for the info - done!